Fix deprecation warning with imagemagick 7.1.

```
WARNING: The convert command is deprecated in IMv7, use "magick" instead of
"convert" or "magick convert"
```
---
--- a/pywal/backends/wal.py
+++ b/pywal/backends/wal.py
@@ -24,9 +24,6 @@
     if shutil.which("magick"):
         return ["magick", "convert"]

-    if shutil.which("convert"):
-        return ["convert"]
-
     logging.error("Imagemagick wasn't found on your system.")
     logging.error("Try another backend. (wal --backend)")
     sys.exit(1)
